pkgs/development/libraries/enchant/2.x.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   src = fetchurl {
```

Comment:
Why is it not `fetchFromGitHub`? https://github.com/AbiWord/enchant/tags has this version. Was there any previous context about not doing this?

Reply:
this fetches a release artifact, not the source code, so I followed that